sumatra/decorators.py

```diff
@@ -13,6 +13,7 @@ def main(parameters, [other_args...]):
 """
 from __future__ import unicode_literals
 
+from builtins import str
```

Comment:
Ah, I see. Apparently I need to look more into the existing Py2/Py3 compatibility infrastructure. Thanks for the explanation!